packages/rum-core/src/domain/view/viewMetrics/trackInteractionToNextPaint.ts

```diff
@@ -72,10 +78,27 @@ export function trackInteractionToNextPaint(
     }
   })
 
+  let telemetryCollected = false
   return {
     getInteractionToNextPaint: (): InteractionToNextPaint | undefined => {
-      // If no INP duration where captured because of the performanceObserver 40ms threshold
-      // but the view interaction count > 0 then report 0
+      if (interactionToNextPaint > 10 * ONE_MINUTE && !telemetryCollected) {
+        telemetryCollected = true
+        addTelemetryDebug('INP > 5 min', {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏The INP is actually > 10min. Maybe use a less specific log message?

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏move this above so we don't need to keep a reference to `inpInteraction`, and it would be simpler to follow